keyboards/hs60/v2/config.h

```diff
@@ -20,8 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0258
+#define VENDOR_ID       0x8968
+#define PRODUCT_ID      0x4853
```

Comment:
I've been told that to use VIA, the ID of the board is derived from `VENDOR_ID` and `PRODUCT_ID`. If you're planning on putting rev1 supported by VIA too, neither of these current `#defines` encode a version number.